src/opm/input/eclipse/Units/UnitSystem.cpp

```diff
@@ -217,7 +217,7 @@ namespace {
         "RM3/DAY",
         "SM3/DAY",  // Should possibly be RM3/DAY
         "M/SEC",    // pipeflow velocity
-        "CPR3/DAY/BARS",
+        "CPM3/D/B", // Transmissibility (e.g., CTFAC)
```

Comment:
Should the type of the constants be changed to const char[9] to avoid this issue in the future?

Reply:
> There seems to be more instances of longer strings below?

Yes.  These will [truncate to eight characters](https://github.com/OPM/opm-common/blob/ae953020b6c50bfbe55f1df982e22f4c520d15ca/opm/io/eclipse/OutputStream.hpp#L396) in the [SMSPEC](https://github.com/OPM/opm-common/blob/ae953020b6c50bfbe55f1df982e22f4c520d15ca/src/opm/io/eclipse/OutputStream.cpp#L682) output, but the result will probably not be what the user expects.  Until we have good replacement strings, however, I prefer to keep these as is.

Reply:
> Should the type of the constants be changed to `const char[9]` to avoid this issue in the future?

It would probably solve half the problem, but likely not all of it.